tests/Functional/Schema/ComparatorTest.php

```diff
@@ -39,7 +39,9 @@ public function testDefaultValueComparison(string $type, $value): void
 
         $onlineTable = $this->schemaManager->listTableDetails('default_value');
 
-        self::assertFalse($this->comparator->diffTable($table, $onlineTable));
+        self::assertFalse(
+            $this->comparator->diffTable($table, $onlineTable, $this->schemaManager->getDatabasePlatform())
```

Comment:
Changes like this effectively stop testing the old implementation. We need to come up with a way of testing both. E.g. each test that should pass with both the new and the old implementations could be duplicated. Later, we can remove the old versions.

Reply:
Done in a0e28045b36445bdfb14d21fb044718a9d2dd7a1